src/Main.java

```diff
@@ -1,5 +1,79 @@
 public class Main {
-    public static void main(String[] args) {
-        System.out.println("Hello world!");
+    public String missingChar(String str, int n) {
+        return str.substring(0, n) + str.substring(n + 1);
+    }
+
+    public int sumDouble(int a, int b) {
+        return a + b;
+    }
+
+    public int intMax(int a, int b, int c) {
+        return intMax(a, b, c);
```

Comment:
This method is just recursion, it doesn't solve anything. Have you tried running it?